public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -51,9 +52,9 @@ export class TableContainer extends PureComponent<Props> {
   }
 
   getTableTitle(dataFrames: DataFrame[] | null, data: DataFrame, i: number) {
-    let title = data.name ? `Table - ${data.name}` : 'Table';
+    let title = data.name ? `${t('explore-table-title', 'Table')} - ${data.name}` : `${t('explore-table-title', 'Table')}`;
```

Comment:
These keys do not match what's in the grafana.json files. You should avoid editing those files manually, and instead run `yarn i18n:extract` to update them. We have automated checks that will prevent merging until this is resolved.

Additionally, we shouldn't manually concatenate values into strings as this limits localisation across different languages. Instead, we can add parameters to the strings:

```ts
let title = data.name
  ? t('explore.table.title-with-name', 'Table - {{name}}', {name: data.name})
  : t('explore.table.title', 'Table');
```

Reply:
@joshhunt  thank you for the feedback, I updated according to it.